letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
I don't think this is a hack. It's only a way to avoid to have an environment. You don't need it for anything. For our architecture your solution doesn't work. Our solution works both for us and for you.